third_party/ibis/ibis_addon/operations.py

```diff
@@ -213,6 +220,12 @@ def sa_format_hashbytes_oracle(translator, expr):
     hash_func = sa.func.standard_hash(compiled_arg, sa.sql.literal_column("'SHA256'"))
     return sa.func.lower(hash_func)
 
+def sa_format_hashbytes_db2(translator, expr):
+    arg, how = expr.op().args
+    compiled_arg = translator.translate(arg)
+    hashfunc = sa.func.hash(compiled_arg,sa.sql.literal_column("2"))
+    return sa.func.hex(hashfunc)
```

Comment:
I think we have to convert to hash to lower case before returning `sa.func.lower(sa.func.hex(hashfunc))`. By default as I have checked it returns hash value in caps.